resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -377,6 +377,25 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: scanImages
```

Comment:
Shouldn't we consider to take the images to scan from the common pipeline environment?

Reply:
I think it would be the next step of this. The first pilot of this will be implemented for Black Duck detect. 
For now it's just a workaround for some specific cases